src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Shell/WindowChromeWorker.cs

```diff
@@ -1378,19 +1133,10 @@ private void _UnhookCustomChrome()
 
         private void _RestoreFrameworkIssueFixups()
         {
-            var rootElement = (FrameworkElement)VisualTreeHelper.GetChild(_window, 0);
+            FrameworkElement rootElement = (FrameworkElement)VisualTreeHelper.GetChild(_window, 0);
```

Comment:
I think we can make similar changes in line 307 as well for consistency reason

Reply:
Agreed, any `var` that goes away is a good thing. Will push that later today. Thanks.

Reply:
@siagupta0202 Adjusted, thanks.